Documentation/bpf.rst

```diff
@@ -4581,12 +4581,22 @@ legacy cBPF:
 
 ..
 
-* **gobpf**
+* **cilium/ebpf**
 
-  gobpf provides go bindings for the bcc framework as well as low-level routines in
-  order to load and use BPF programs from ELF files.
+  `cilium/ebpf` (ebpf-go) is a pure Go library that provides utilities for
+  loading, compiling, and debugging eBPF programs. It has minimal external
+  dependencies and is intended to be used in long-running processes.
 
-  https://github.com/iovisor/gobpf
+  Its `bpf2go` utility automates away compiling eBPF C programs and embedding
+  them into Go binaries.
+
+  It implements bpf_link for attaching programs to various kernel hooks,
```

Comment:
`BPF links`? I don't know if `bpf_link` is an “official” denomination (or if there is one at all). It's also not mentioned anywhere else on the page at the moment, so not sure if it's worth mentioning or if it may confuse readers.

Reply:
I don't know, I saw it pop up in some early patches and one of your tweets, so I figured it was an accurate term. https://twitter.com/qeole/status/1262775391619551234

Omitting it may be better, agreed.